Source/WebKit/UIProcess/ProcessAssertion.h

```diff
@@ -116,6 +116,7 @@ class ProcessAssertion : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<
     Function<void()> m_prepareForInvalidationHandler;
     Function<void()> m_invalidationHandler;
 #if USE(EXTENSIONKIT)
+    static Lock s_capabilityLock;
     std::optional<AssertionCapability> m_capability;
     RetainPtr<_SEGrant> m_grant;
```

Comment:
Should probably use `WTF_GUARDED_BY_LOCK` for this one and other members guarded by the lock to have the compiler verify we don't have threading bugs.